pootle/apps/pootle_fs/state.py

```diff
@@ -0,0 +1,311 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from collections import OrderedDict
+
+from django.utils.functional import cached_property
+from django.utils.lru_cache import lru_cache
+
+from pootle.core.state import ItemState, State
+from pootle_store.models import FILE_WINS, POOTLE_WINS
+
+from .resources import FSProjectStateResources
+
+FS_STATE = OrderedDict()
+FS_STATE["conflict"] = {
+    "title": "Conflicts",
+    "description": "Both Pootle Store and file in filesystem have changed"}
+FS_STATE["conflict_untracked"] = {
+    "title": "Untracked conflicts",
+    "description": (
+        "Newly created files in the filesystem matching newly created Stores "
+        "in Pootle")}
+FS_STATE["pootle_ahead"] = {
+    "title": "Changed in Pootle",
+    "description": "Stores that have changed in Pootle"}
+FS_STATE["pootle_untracked"] = {
+    "title": "Untracked Stores",
+    "description": "Newly created Stores in Pootle"}
+FS_STATE["pootle_staged"] = {
+    "title": "Added in Pootle",
+    "description": (
+        "Stores that have been added in Pootle and are now being tracked")}
+FS_STATE["pootle_removed"] = {
+    "title": "Removed from Pootle",
+    "description": "Stores that have been removed from Pootle"}
+FS_STATE["fs_staged"] = {
```

Comment:
Nit: Use the same order as for `pootle_`, so `*_ahead`, `*_untracked`, `*_staged`, `*_removed`.